### DISCUSSION TITLE
de/2-00-learning-pathway/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-ai#49] [General]
# de/2-00-learning-pathway/

https://ai.lernos.org/de/2-00-learning-pathway/

**Comment by Marianne910** [comment]
zu dem verlinkten Beispiel - einen KI-unterstützten Blog-/LinkedIn-Beitrag zu erstellen und zu veröffentlichen  -@Matthias Seiller - ist die Beschreibung des Erstellprozesses für den Post öffentlich zugänglich, d.h. wie du aus dem Interview mittels KI den Post generiert hast?

**Reply by simondueckert** [reply]
Du hast einen Matthias Platzer erwähnt, du meintest Matze, der bekommt das so nicht mit. Vielleicht schreibst du ihn dazu [auf Linkedin](https://www.linkedin.com/in/matthias-seiller-056170206/) an.

**Reply by Marianne910** [reply]
ich persönlich brauche die Info ja nicht  - aber zur Verbesserung des Leitfadens: wäre es nicht sinnvoller, wenn das Beispiel auf das hier verwiesen wird den Erstellprozess der mit KI generierten Posts beschreibt und nicht nur auf das Ergebnis in Form eines Post verlinkt? 
(PS Mattias Seiler hat diesen Erstellprozess an anderer Stelle beschrieben wie aus den Kommentaren zu seinem Post erkennbar wird - da habe ich schon angefragt ob der Erstellprozess irgendwo dokumentiert ist, bislang keine Antwort und für mich damit erledigt)
@matthias - sorry falscher Bezug gemeint war Mattias Seiler

**Reply by simondueckert** [reply]
Ich spreche Matze kommende Woche mal an. Sinnvoll wäre das natürlich, soweit ich weiß, gibt es bisher aber keine Doku des Erstellprozesses.

**Reply by maki-amw** [reply]
@simondueckert gibt es hierzu noch weitere Infos zum Erstellorozess? Oder sollten wir hier ggf. ein alternatives Beispiel finden?

**Comment by DoSchu-cowoxu** [comment]
In der Erstellung eines Lernangebots auf der Basis des lernOS KI Leitfadens wurde diese Seite als zu wenig inhaltlich hinführend empfunden. Besonders im PDF haben wir ja keine Seitennavigation als Zusatzinfo im Blick. Zusätzlich zur Grafik könne dieser Text verwendet werden: 

start

Das sind die Etappen, die du im Lernpfad durchlaufen wirst:

Kata 0: Vorbereiten & Kennenlernen
Kata 1: Bewusstsein schaffen
Kata 2: Einstieg mit der KI
Kata 3: KI als DialogpartnerIn
Kata 4: Reflexion der Lernergebnisse
Kata 5: Anwendungsfelder am eigenen Arbeitsplatz
Kata 6: Vertiefung Deines Lernziels
Kata 7: Zusammenarbeit mit KI
Kata 8: Reflexion der Lernergebnisse
Kata 9: KI als Träumer, Halluzinator oder Lügner
Kata 10 Souverän prompten Teil 1
Kata 11 Souverän prompten Teil 2 
Kata 12 Review und Lessons Learned

Zu jeder Etappe gibt der Leitfaden Hinweise für die inhaltliche Erarbeitung des jeweiligen Themas sowie konkrete Übungsaufgaben.

ende

**Reply by maki-amw** [reply]
Vielen Dank für den Hinweis @DoSchu-cowoxu ! Das arbeite ich gleich einmal entsprechend mit ein.

**Comment by Flong-ki** [comment]
Halllo!
Die pdf-Version ist gerade nicht zugänglich. Oder gibt es da einen Trick?

**Reply by simondueckert** [reply]
Sorry, das war ein Bug unsererseits, jetzt geht der Link wieder.
